modules/dns_nsupdate/dns_nsupdate_main.rb

```diff
@@ -69,6 +80,10 @@ def remove_ptr_record(ip)
       do_remove(ip, "PTR")
     end
 
+    def remove_cname_record(fqdn_alias)
```

Comment:
I agree `fqdn_alias` isn't the best variable name, but `name` is better then anything containing `alias`? I would like something more meaningful then `name`, since `alias` is a reserved keyword